### DISCUSSION TITLE
Currently, Emotion is supported, but is there a possibility that it might be deprecated in the future

### DISCUSSION THREAD

**Discussion by jest-won** [mantinedev/mantine#7014] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.9.0

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

All

### Describe the bug

Currently, Emotion is supported, but is there a possibility that it might be deprecated in the future?
Is Vanilla Extract the right choice for using CSS in JS?

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
CSS in JS is not the right choice. If you want your app to be future-proof, use CSS modules.
